### PR TITLE
made product pages for each item available for sale, which includes c…

### PR DESCRIPTION
**Pull request by cartershadd**
…checkboxes for customization

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -14,7 +15,8 @@ function App() {
                 <Switch>
                     <Route exact path={"/"} component={Home}/>
                     <Route exact path={"/shop"} component={MainShopPage}/>
-                    <Route exact path={'/item'} component={Item}/>
+                    <Route exact path={'/cloud'} component={Cloud}/>
+                    <Route exact path={'/unicorn'} component={Unicorn}/>
```
Comment (nburn42) [inline src/App.js:19]: maybe these could be `/shop/unicorn`
It doesn't matter that much but I think the `/shop/unicorn` format is more standard
and you could expand it in the future to `/shop/%%anyproduct%%` and make it dynamic

File: src/Pages/Cloud.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import Form from 'react-bootstrap/Form';
+
+const Unicorn = () => {
+    return (
+        <Container fluid>
+            <Header/>
+            <Row xl={3}>
+                <Col>
+
+                    <Image src={cloud1} thumbnail/> </Col>
```
Comment (nburn42) [inline src/Pages/Cloud.js:20]: move the col closing tab to its own line

File: src/Pages/Cloud.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                            id={1}
+                            label={`Theatre Chase`}
+                        />
+                        <br />
+                        <Form.Check
+                            custom
+                            type="checkbox"
+                            id={1}
+                            label={`Thundercloud`}
+                        />
```
Comment (nburn42) [inline src/Pages/Cloud.js:63]: These ids should be unique

File: src/Pages/Unicorn.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                            id={1}
+                            label={`Theatre Chase`}
+                        />
+                        <br />
+                        <Form.Check
+                            custom
+                            type="checkbox"
+                            id={1}
+                            label={`Thundercloud`}
+                        />
```
Comment (nburn42) [inline src/Pages/Unicorn.js:63]: Ids should be unique

**Review by nburn42 — APPROVED** [review]
Great Job

File: src/Pages/Cloud.js (hunk 3 above)
Comment (cartershadd) [reply src/Pages/Cloud.js:63]: has been fixed, thanks for catching it

File: src/App.js (hunk 1 above)
Comment (cartershadd) [reply src/App.js:19]: i am going to use that idea.

**Pull request merged**